cfme/services/service_catalogs/ui.py

```diff
@@ -88,28 +108,18 @@ def is_displayed(self):
         )
 
 
-class ServiceCatalogs(Updateable, Pretty, Navigatable):
-
-    def __init__(self, catalog=None, name=None, stack_data=None,
-                 dialog_values=None, ansible_dialog_values=None, appliance=None):
-        Navigatable.__init__(self, appliance=appliance)
-        self.catalog = catalog
-        self.name = name
-        self.stack_data = stack_data
-        self.dialog_values = dialog_values
-        self.ansible_dialog_values = ansible_dialog_values
-
-    def order(self):
-        view = navigate_to(self, 'Order')
-        if self.stack_data:
-            view.fill(self.stack_data)
-        if self.dialog_values:
-            view.fill(self.dialog_values)
-        if self.ansible_dialog_values:
-            view.fill(self.ansible_dialog_values)
-        view.submit_button.click()
-        # Request page is displayed after this hence not asserting for view
-        view.flash.assert_success_message("Order Request was Submitted")
+@ServiceCatalogs.order.external_implementation_for(ViaUI)
+def order(self):
+    view = navigate_to(self, 'Order')
+    if self.stack_data:
+        view.fill(self.stack_data)
+    if self.dialog_values:
+        view.fill(self.dialog_values)
+    if self.ansible_dialog_values:
+        view.fill(self.ansible_dialog_values)
+    view.submit_button.click()
+    # Request page is displayed after this hence not asserting for view
```

Comment:
You can import this view and use create_view() to create an instance of it in order to call is_displayed.